docs/guides/location-service/fragments/setting-up-your-app-ios.md

```diff
@@ -0,0 +1,249 @@
+## Overview
+
+[Amazon Location Service](http://aws.amazon.com/location) is a geolocation service that lets you add location information to your applications. With Amazon Location Service, you can build applications that provide maps and points of interest, track resources, trigger actions based on location, and convert street addresses into geographic coordinates. This guide provides you with the following information:
```

Comment:
There seems to be a lot of duplicated content in this fragment. Is it possible to get the English-language content into a common root, and then use templates for the platform-specific code snippets?

Reply:
:thumbsup: Good suggestion for later.